setup.py

```diff
@@ -74,7 +74,7 @@ def run_tests(self):
         "redis": ["aioredis"],
         "arctic": ["arctic"],
         "zmq": ["pyzmq"],
-        "mongo": ["motor"],
+        "mongo": ["bson", "motor"],
```

Comment:
this doesnt need to be manually installed. motor includes it. Try creating a virtualenv, and installing only motor - you'll find that bson is available for import. Its built into pymongo/motor

Reply:
You are right, I can read from PyMongo packages distribution:
https://github.com/mongodb/mongo-python-driver#installation

> Do not install the "bson" package from pypi. PyMongo comes with its own bson package; doing "easy_install bson" installs a third-party package that is incompatible with PyMongo.